os/arch/arm/src/stm32/stm32_otgfshost.c

```diff
@@ -673,6 +706,7 @@ static inline void stm32_chan_freeall(FAR struct stm32_usbhost_s *priv)
 
 static void stm32_chan_configure(FAR struct stm32_usbhost_s *priv, int chidx)
 {
+	FAR struct stm32_chan_s *chan = &priv->chan[chidx];
```

Comment:
No need to add checks for input variables?

Reply:
Done